src/test/scala/chiselTests/Vec.scala

```diff
@@ -2,16 +2,81 @@
 
 package chiselTests
 
-import org.scalatest._
-import org.scalatest.prop._
-
 import chisel3._
+import chisel3.core.Binding.BindingException
 import chisel3.testers.BasicTester
 import chisel3.util._
-//import chisel3.core.ExplicitCompileOptions.Strict
+import org.scalacheck.Shrink
+
+class LitTesterMod( vecSize : Int ) extends Module {
+  val io = IO(new Bundle {
+    val out = Output(Vec( vecSize, UInt() ))
+  })
+  io.out := Vec( vecSize, 0.U )
+}
+
+class RegTesterMod( vecSize : Int ) extends Module {
+  val io = IO(new Bundle {
+    val in = Input(Vec( vecSize, UInt() ))
+    val out = Output(Vec( vecSize, UInt() ))
+  })
+  val vecReg = Reg( init = Vec( vecSize, 0.U ), next = io.in )
+  io.out := vecReg
+}
+
+class IOTesterMod( vecSize : Int ) extends Module {
+  val io = IO(new Bundle {
+    val in = Input(Vec( vecSize, UInt() ))
+    val out = Output(Vec( vecSize, UInt() ))
+  })
+  io.out := io.in
+}
+
+class LitTester(w: Int, values: List[Int] ) extends BasicTester {
```

Comment:
You should style your arguments consistently. I think we typically have no spaces after the begin parentheses or before the end parentheses, like `class MyClass(myVal: Type)`. But this like is just ridiculous, where you mix space and non-space...